packages/ui/src/forum/types/ForumCategory.ts

```diff
@@ -0,0 +1,15 @@
+import { ForumCategoryFieldsFragment } from '@/forum/queries/__generated__/forum.generated'
+
+export interface ForumCategory {
+  id: string
+  title: string
+  description: string
+}
+
+export const asForumCategory = (fields: ForumCategoryFieldsFragment): ForumCategory => {
+  return {
+    id: fields.id,
+    title: fields.title,
+    description: fields.description,
+  }
+}
```

Comment:
```suggestion
export const asForumCategory = (fields: ForumCategoryFieldsFragment): ForumCategory => ({
    id: fields.id,
    title: fields.title,
    description: fields.description,
})
```

Reply:
Not a problem, but I think it looks better 👀

Reply:
Yep, I'll update that.